### PR TITLE
feat: logo component is created

### PR DESCRIPTION
**Pull request by OkancanOkan**
### SUMMARY
New DVT-LOGO component is created

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { styled } from '@superset-ui/core';
+
+const StyledDvtLogo = styled.div`
+  display: flex;
+  width: 140px;
+  height: 29px;
+  justify-content: flex-end;
+  align-items: flex-start;
+  gap: 9px;
+  flex-shrink: 0;
```
Comment (omerfbolat) [inline superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:10]: flex-shrink: 0; ?? bu flex divler kullanmıyor.

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+`;
+
+const DvtTitle = styled.div`
+  color: ${({ theme }) => theme.colors.dvt.text.bold};
+  font-family: Inter;
```
Comment (omerfbolat) [inline superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:15]: font-family: Inter; kaldırmalı. Body default olarak kullanıyor

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  font-size: 23.2px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 125%; /* 29px */
+  letter-spacing: -0.29px;
+`;
+
+const DvtLogoImg = styled.a`
```
Comment (omerfbolat) [inline superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:23]: styled.a aslında (Link) olmalı

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { styled } from '@superset-ui/core';
+
+const StyledDvtLogo = styled.div`
+  display: flex;
+  width: 140px;
+  height: 29px;
+  justify-content: flex-end;
```
Comment (omerfbolat) [inline superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:7]: justify-content: flex-end; kalmamalı. Sol logo sabit kalmalı

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  align-items: flex-start;
```
Comment (omerfbolat) [inline superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:8]: align-items: flex-start; > flex-start her zaman default kullanıyor. kaldırmalı

File: superset-frontend/src/components/DvtLogo/dvt-logo.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import DvtLogo, { DvtLogoProps } from './index';
```
Comment (omerfbolat) [inline superset-frontend/src/components/DvtLogo/dvt-logo.stories.tsx:2]: import DvtLogo, { DvtLogoProps } from './index';

'./index' olmamalı > '.' olmalı

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx (hunk 1 above)
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:10]: completed

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx (hunk 2 above)
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:15]: completed

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  font-style: normal;
+  font-weight: 700;
+  line-height: 125%; /* 29px */
+  letter-spacing: -0.29px;
+`;
+
+const DvtLogoImg = styled.a`
+  width: 27.84px;
+  height: 27.84px;
+  flex-shrink: 0;
```
[the remark this answers is not in the document]
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:26]: tamamlandı

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx (hunk 5 above)
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:8]: completed

File: superset-frontend/src/components/DvtLogo/dvt-logo.stories.tsx (hunk 6 above)
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.stories.tsx:2]: completed

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx (hunk 3 above)
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:23]: completed

File: superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx (hunk 4 above)
Comment (OkancanOkan) [reply superset-frontend/src/components/DvtLogo/dvt-logo.module.tsx:7]: bu da tamamdır

**Pull request closed without merging**